File: Javascript/Uciv.js
// Let nr1 and nr2 be two numbers. Find out the last digit of the sum of a and b.
nr1 = parseInt(prompt("What is the first number?\n-"));
console.log("The first number is "+nr1.toString()+".");
nr2 = parseInt(prompt("What is the second number?\n-"));
console.log("The first number is "+nr2.toString()+".");
sum = nr1 + nr2;
console.log("The sum of the two numbers is "+sum.toString()+".");
lastDigit = sum % 10;
console.log("The last digit of the sum is "+lastDigit.toString()+".");

File: Python/Uciv.py
# Let a and b be two numbers,and c be their sum  Find out the last digit of c.
a = int(input("What is the first number?\n-"))
print("The first number is " + str(a) + ".")

b = int(input("What is the second number?\n-"))
print("The second number is " + str(b) + ".")

c = a + b
print("Their sum is " + str(c) + ".")

d = c % 10
print("The last digit of that sum is " + str(d) + ".")
